lib/PHPCfg/Parser.php

```diff
@@ -1028,7 +1029,12 @@ protected function parseExpr_Exit(Expr\Exit_ $expr)
             $e = $this->readVariable($this->parseExprNode($expr->expr));
         }
 
-        return new Op\Expr\Exit_($e, $this->mapAttributes($expr));
+        $this->block->children[] = new Op\Terminal\Exit_($e, $this->mapAttributes($expr));
+        // Dump everything after the exit
+        $this->block = new Block();
+        $this->block->dead = true;
+
+        return new Op\Terminal\Exit_($e, $this->mapAttributes($expr));
```

Comment:
This should be `return new Literal(1);` It needs to be an Operand and any dummy value would do. Using `1` is what PHP does internally.